validator/src/main/java/com/google/daq/mqtt/sequencer/SequenceBase.java

```diff
@@ -246,7 +246,8 @@ public class SequenceBase {
   private static final String FAKE_DEVICE_ID = "TAP-1";
   private static final String NO_EXTRA_DETAIL = "no logs";
   private static final Duration DEFAULT_WAIT_TIME = Duration.ofSeconds(10);
-  private static final Duration CONFIG_WAIT_TIME = Duration.ofSeconds(30);
+  // TODO Config wait time increased for endpoint_failure_and_restart test
```

Comment:
TODO?  Comment is describing the change not what should be done... ?

Reply:
Removed, I was looking to increase the timeout for a specific test and forgot to remove it. This will increase the overall test duration in case of a failure.